ci-operator/step-registry/cucushift/installer/check/ibmcloud/cucushift-installer-check-ibmcloud-commands.sh

```diff
@@ -12,8 +12,49 @@ function checkProviderType {
     providerType=$(yq-go r ${platformStatusFile} 'status.platformStatus.ibmcloud.providerType')
     if [[ ${providerType} != "VPC" ]]; then
         echo "ERROR: ${providerType} is not expected providerType [VPC]!!"
+        check_result=1
```

Comment:
If `check_result` is defined in main function, then should update it in main function per the return value from this function.